frontend/server/src/main/java/com/amazonaws/ml/mms/util/codec/ModelRequestEncoder.java

```diff
@@ -27,6 +27,10 @@
 @ChannelHandler.Sharable
 public class ModelRequestEncoder extends MessageToByteEncoder<BaseModelRequest> {
 
+    public ModelRequestEncoder(boolean flag) {
+        //super(flag);
```

Comment:
This may have been mistakenly commented out.

Reply:
this was for testing, please review next rev